ui/src/pages/Board.scss

```diff
@@ -66,15 +66,14 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  transform: scale(1.02);
```

Comment:
This is to eliminate gaps between tiles that make it look weird. It would be better to regenerate the svg files with a small gap between the outer border and the inner pattern. There is a little bit of "bleeding" underneath the dark gray border that I'd love to get rid of. Not sure how you got created these svgs though.
![image](https://github.com/user-attachments/assets/90127437-4816-46cf-b471-c7186e42842d)